chapter4/e2-recursion.js

```diff
@@ -0,0 +1,6 @@
+function isEven(n) {
```

Comment:
This function is almost right but doesn't actually answer the prompt. At a lot of places with strict rubrics this is enough to fail the interview.